docker-compose.yaml

```diff
@@ -152,6 +155,6 @@ services:
       - 80:80
 
 volumes:
-
     minio-data:
     database_volume:
+    ray-huggingface-cache:
```

Comment:
Is ‘ray-huggingface-cache‘ the one that’s commented out above now?

Reply:
Yes, afaict.